pkg/osbuild/dnf_mark_stage.go

```diff
@@ -0,0 +1,56 @@
+package osbuild
+
+import (
+	"github.com/osbuild/images/pkg/rpmmd"
+)
+
+type DNFMarkStagePackageOptions struct {
+	Name  string `json:"name"`
+	Mark  string `json:"mark"`
+	Group string `json:"group,omitempty"`
```

Comment:
I think that the latest version of the `osbuild` PR doesn't have `group` in the stage options.